.github/workflows/build.yml

```diff
@@ -113,6 +113,55 @@ jobs:
         git -C taps/ push
 
 
+  build_unix_arm:
+      needs: prepare
+      runs-on: ubuntu-18.04  # Standalone executable should be built on minimum supported OS
```

Comment:
does this version matter?

Reply:
Not really, but I've done it to match the version used by the build_unix step

Reply:
```suggestion
      runs-on: ubuntu-latest
```